app/helpers/application_helper.rb

```diff
@@ -178,9 +178,12 @@ def manifest_url(resource)
   end
 
   # Generate the path for the Universal Viewer iframe @src attribute
+  # @param [Valkyrie::Resource]
   # @return [String]
   def universal_viewer_path(resource)
-    "/uv/uv#?manifest=#{manifest_url(resource)}&config=#{root_url}/uv/uv_config.json"
+    config_path = resource.decorate.downloadable? ? "uv_config.json" : "uv_config_downloads_disabled.json"
```

Comment:
I think we should show the regular viewer to users who are `admin` or `staff` — e.g., so they can download a PDF or page images to fulfill a user request.